DependencyInjection/Configuration.php

```diff
@@ -69,6 +69,12 @@ public function getConfigTreeBuilder()
                 ->scalarNode('tenant_href')->end()
                 ->scalarNode('default_application_name')->end()
                 ->scalarNode('default_application_href')->end()
+                ->arrayNode('groups')
```

Comment:
This is potentially confusing. I would suggest putting these under a registries key, making it clear at least where the entered data will be going.